index.js

```diff
@@ -49,6 +49,18 @@ function serve(root, opts) {
     // response is already handled
     if (this.body != null || this.status != 404) return;
 
-    yield send(this, this.path, opts);
+    yield send(this, formatPath(this.path), opts);
   };
+
+  // Format the path to serve static file servers
+  // and not require a trailing slash for directories,
+  // so that you can do both `/directory` and `/directory/`
+  function formatPath(path) {
+    if (opts.format) {
+      var trailingSlash = '/' == path[path.length - 1];
```

Comment:
Good catch, I finally did the pull request at the koa-send level : https://github.com/koajs/send/pull/28